generative_data_prep/__main__.py

```diff
@@ -290,13 +288,36 @@ def get_categories(categories_path: str):
     return category_to_id
 
 
+def create_Logger(log_file_path: str, output_dir: str):
+    """If log_file_path is defined then return it, otherwise return output_dir/logs.log.
+
+    Args:
+        log_file_path: The input log_file_path flag.
+        output_dir: The output directory to default to if log_file_path is None.
+    """
+    if log_file_path is None:
+        log_file_path = os.path.join(output_dir, "logs.log")
+
+    formatter = logging.Formatter("%(message)s")
+    file_handler = logging.FileHandler(log_file_path)
+    file_handler.setFormatter(formatter)
+    file_handler.setLevel(logging.DEBUG)
+    # Add the file handler to the Logger()
+    logger.add_handler(file_handler)
+    logger.log_git_commit_hash()
+    logger.log_current_datetime()
```

Comment:
i recommend using `yaml` file for logging configuration:

https://docs.python.org/3/library/logging.config.html#configuration-file-format

Reply:
Done! Used an ini file